### PR TITLE
Improve execution

### PR DESCRIPTION
**Pull request by akoniar**
Resolve issues #10 and #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,117 @@
...
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker-redhat) boosters
+* Secured Mission
+    * Eclipse Vert.x [community](https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster) and [product](https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster-redhat) boosters
+    * Node.js [community](https://github.com/bucharest-gold/nodejs-rest-http-secured) and [product](https://github.com/bucharest-gold/nodejs-rest-http-secured-redhat) boosters
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-http-secured-booster) and [product](https://github.com/snowdrop/spring-boot-http-secured-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-secured) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-secured-redhat) boosters
```
Comment (Ladicek) [inline README.md]: Why do we need this list?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,117 @@
...
 ```
-export HTTP_BOOSTER_URL="url for http booster"
-export CRUD_BOOSTER_URL="url for crud booster"
-export CIRCUIT_BREAKER_BOOSTER_URL="url for circuit breaker booster"
-export CONFIG_MAP_BOOSTER_URL="url for config map booster"
-export HEALTH_CHECK_BOOSTER_URL="url for health check booster"
-export SECURED_HTTP_BOOSTER_URL="url for secured http booster"
 
-export CONFIG_MAP_BOOSTER_VALUE="Hello, %s from a ConfigMap !"
+## Run tests for all booster with the environmental variables
```
Comment (Ladicek) [inline README.md]: I'm not interested in this. We can mention it towards the end of the README, but this is not the primary usecase.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,117 @@
...
+Is necessary to set URL address of boosters and runtime of the boosters. Optionally you can set config map value and user credentials for secured booster
```
Comment (Ladicek) [inline README.md]: I'm not interested in changing the optional parameters. We can mention it towards the end of the README, but this is not the primary usecase.

File: README.md (hunk 1 above)
Comment (akoniar) [reply README.md]: It is for better searching of boosters. I can transfer this info to the wiki page

**Pull request merged**